src/utils/internal.ts

```diff
@@ -262,6 +269,13 @@ export function createClaim(
         id: cddId as string,
       };
     }
+    case ClaimType.Custom: {
+      return {
+        type,
+        customClaimTypeId: customClaimTypeId as BigNumber,
```

Comment:
It seems like non null assertion works on the type and looks more natural to me. OK to leave if you prefer `as BigNumber` or if we need it for some reason though
```suggestion
        customClaimTypeId: customClaimTypeId!,
```

Also, should this throw an internal error if `customClaimTypeId` is falsy here? TS isn't going to stop this being undefined.